build/taskctl/tasks.yaml

```diff
@@ -9,7 +9,7 @@ tasks:
     context: powershell-python
     description: Perform YAML linting
     command:
-      - Invoke-YamlLint
+      - Invoke-YamlLint -FailOnWarnings $False
```

Comment:
We now have the ability to pass in this new argument so the pipeline doesn't fail if warnings are found. This can be removed once warnings are resolved .